website/source/docs/operating-a-job/update-strategies/blue-green-and-canary-deployments.html.md

```diff
@@ -17,136 +16,438 @@ organizations prefer to put a "canary" build into production or utilize a
 rollout to production while minimizing downtime.
 
+## Blue/Green Deployments
+
 Blue/Green deployments have several other names including Red/Black or A/B, but
 the concept is generally the same. In a blue/green deployment, there are two
 application versions. Only one application version is active at a time, except
 during the transition phase from one version to the next. The term "active"
 tends to mean "receiving traffic" or "in service".
 
-Imagine a hypothetical API server which has ten instances deployed to production
-at version 1.3, and we want to safely upgrade to version 1.4. After the new
-version has been approved to production, we may want to do a small rollout. In
-the event of failure, we can quickly rollback to 1.3.
+Imagine a hypothetical API server which has five instances deployed to
+production at version 1.3, and we want to safely upgrade to version 1.4. We want
+to create five new instances at version 1.4 and in the case that they are
+operating correctly we want to promote them and take down the five versions
+running 1.3. In the event of failure, we can quickly rollback to 1.3.
 
-To start, version 1.3 is considered the active set and version 1.4 is the
-desired set. Here is a sample job file which models the transition from version
-1.3 to version 1.4 using a blue/green deployment.
+To start, we examine our job which is running in production:
 
 ```hcl
 job "docs" {
-  datacenters = ["dc1"]
+  ...
 
-  group "api-green" {
-    count = 10
+  group "api" {
+    count = 5
 
     task "api-server" {
       driver = "docker"
 
-      config {
-        image = "api-server:1.3"
+      update {
+        max_parallel     = 1
+        canary           = 5
+        min_healthy_time = "30s"
+        healthy_deadline = "10m"
+        auto_revert      = true
       }
-    }
-  }
-
-  group "api-blue" {
-    count = 0
-
-    task "api-server" {
-      driver = "docker"
 
       config {
-        image = "api-server:1.4"
+        image = "api-server:1.3"
       }
     }
   }
 }
 ```
 
-It is clear that the active group is "api-green" since it has a non-zero count.
-To transition to v1.4 (api-blue), we increase the count of api-blue to match
-that of api-green.
+We see that it has an `update` stanza that has the `canary` equal to the desired
+count. This is what allows us to easily model blue/green deployments. When we
+change the job to run the "api-server:1.4" image, Nomad will create 5 new
+allocations without touching the original "api-server:1.3" allocations. Below we
+can see how this works by changing the image to run the new version:
 
 ```diff
 @@ -2,6 +2,8 @@ job "docs" {
- group "api-blue" {
--  count = 0
-+  count = 10
-
-   task "api-server" {
-     driver = "docker"
+  group "api" {
+    task "api-server" {
+      config {
+-       image = "api-server:1.3"
++       image = "api-server:1.4"
 ```
 
 Next we plan and run these changes:
 
 ```shell
 $ nomad plan docs.nomad
```

Comment:
Since you're including output here, change the type from `shell` to `text`. Sometimes the highlighter gets weird otherwise